lib/functions/index.js

```diff
@@ -21,6 +23,8 @@ module.exports = {
   every,
   filter,
   first,
+  single,
+  last,
```

Comment:
Although undocumented, these are all in alphabetic order, it would be nice to keep this pattern.